### DISCUSSION TITLE
Cycling sizes repeated commands not working for me

### DISCUSSION THREAD

**Discussion by szinn** [rxhanson/RectanglePro-Community#626] [Q&A]
I reinstalled MacOS and am getting everything set up. Unfortunately I did not export my configuration for RectanglePro.

I used to be able to repeat the "first third" keystroke and it would cycle through 1/3, 1/2, 2/3. Currently it cycles through first third, middle third, last third.

I have the General setting Repeated Commands set to "cycle sizes on half actions" and 1/3 and 2/3 selected.

What am I missing?

**Comment by szinn** [comment]
I tried restoring ~/Library/Application Support/Rectangle Pro from a recent backup but I just get the same settings as before the restore.

Is the configuration stored somewhere else?

**Reply by rxhanson** [reply]
The configuration is at 
```
~/Library/Preferences/com.knollsoft.Hookshot.plist
```
(confusing, but Hookshot is the original name of Rectangle Pro)

**Comment by rxhanson** [comment]
Cycling 1/2, 2/3, 1/3 is for half actions, like left half, and I suspect that what you had configured previously. `cycle sizes on half actions` will give you that, so you should be all set to just use left/right half to cycle through those sizes.

The thirds cycling between first, middle, and last third is how the thirds commands have pretty much always behaved.

If you were looking to configure specifically for that order of cycling: 1/3, 1/2, 2/3, then you could create a custom size and position that would behave that way.  Here's a discussion on how to do this: https://github.com/rxhanson/RectanglePro-Community/discussions/5

**Comment by szinn** [comment]
Thanks!

I tried restoring the plist from a backup but the previous wrong configuration still seems to be there.

In any case, moving to the half commands did what I wanted so I'm back in my happy place again.
